packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,162 @@
+const addApplePaySessionToChrome = () => {
+    class ApplePaySessionCustomerStep implements ApplePaySession {
+        version: number;
+        paymentRequest: ApplePayJS.ApplePayPaymentRequest;
+
+        constructor(version, paymentRequest) {
+            this.version = version;
+            this.paymentRequest = paymentRequest;
+        }        
+        addEventListener(_type: string, _callback: EventListenerOrEventListenerObject | null, _options?: boolean | AddEventListenerOptions | undefined): void {
+            console.log('addEventListener', _type, _callback, _options);
+        }
+        dispatchEvent(_event: Event): boolean {
+            console.log('dispatchEvent', _event);
+            return true
+        }
+        removeEventListener(_type: string, _callback: EventListenerOrEventListenerObject | null, _options?: boolean | EventListenerOptions | undefined): void {
+            console.log('removeEventListener', _type, _callback, _options);
+        }
+
+        oncancel: (event: ApplePayJS.Event) => void = () => { 
+            console.log('oncancel');
+        }
+
+        onpaymentauthorized: (event: ApplePayJS.ApplePayPaymentAuthorizedEvent) => void = () => {
+            console.log('onpaymentauthorized');
+        }
+
+        onpaymentmethodselected: (event: ApplePayJS.ApplePayPaymentMethodSelectedEvent) => void = () => {
+            console.log('onpaymentmethodselected');
+        }
+
+        onshippingcontactselected: (event: ApplePayJS.ApplePayShippingContactSelectedEvent) => void = () => { 
+            console.log('onshippingcontactselected');
+        }
+
+        onshippingmethodselected: (event: ApplePayJS.ApplePayShippingMethodSelectedEvent) => void  = () => { 
+            console.log('onshippingmethodselected');
+        }
+
+        onvalidatemerchant: (event: ApplePayJS.ApplePayValidateMerchantEvent) => void  = () => { 
+            console.log('onvalidatemerchant');
+        }
+
+        abort(): void {
+            console.log('abort')
+        }
+
+        STATUS_SUCCESS = 1;
```

Comment:
Same here, plus:
```suggestion
        static STATUS_SUCCESS = 1;
```

Reply:
move to the top as well as added as a static variable